src/source-updater.js

```diff
@@ -428,12 +428,23 @@ export default class SourceUpdater extends videojs.EventTarget {
    * @return {boolean}
    *         if changeType can be called.
    */
-  canChangeType() {
+  static canChangeType() {
     return window.SourceBuffer &&
         window.SourceBuffer.prototype &&
         typeof window.SourceBuffer.prototype.changeType === 'function';
   }
 
+  /**
+   * Whether or not the changeType function is supported
+   * on our SourceBuffers.
+   *
+   * @return {boolean}
+   *         if changeType can be called.
+   */
+  canChangeType() {
```

Comment:
Is it worth having this too vs just using the static function?

Reply:
This method is used internally and makes its usage slightly easier.